app/src/main/java/org/simple/clinic/login/pin/LoginPinScreenController.kt

```diff
@@ -179,6 +180,10 @@ class LoginPinScreenController @Inject constructor(
 
   private fun backClicks(events: Observable<UiEvent>): Observable<UiChange> {
     return events.ofType<PinBackClicked>()
-        .map { { ui: Ui -> ui.goBackToLoginPhoneScreen() } }
+        .flatMapCompletable {
+          userSession.clearLoggedInUser()
+              .andThen(userSession.saveOngoingLoginEntry(OngoingLoginEntry(userId = null)))
```

Comment:
Hrm, I think we should add a `clearOngoingLoginEntry()` function instead of exposing what-is-considered-as-empty. This has happened once in https://github.com/simpledotorg/simple-android/pull/204.